bambora-client.gemspec

```diff
@@ -6,8 +6,8 @@ require 'bambora/client/version'
 Gem::Specification.new do |spec|
   spec.name          = 'bambora-client'
   spec.version       = Bambora::Client::VERSION
-  spec.authors       = ['Cassidy K']
-  spec.email         = ['hello@cassidy.codes', 'tech@himama.com']
+  spec.authors       = ['HiMama']
+  spec.email         = ['tech@himama.com']
```

Comment:
Removing former employee from Ruby gems page:

<img width="557" alt="image" src="https://github.com/HiMamaInc/bambora-client/assets/61441409/9a6c2f11-d3e9-4c38-9638-09ddc134538a">

(If someone feels like automating all the contributors, that's something we can look into.)